.github/workflows/release-new-release-published.yml

```diff
@@ -8,14 +8,24 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: 'Notify to Slack when a new release is published'
+      - name: 'Notify to release channel'
         if: ${{ !contains(github.event.release.tag_name, 'wc-beta-tester') }}
```

Comment:
Is there a reason to apply the `if` condition per step instead of at [the jobs level](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idif)? That way the full job is skipped if the condition is not met, instead of having to check per step.

Reply:
You are absolutely right! Fixed in 4c3fd402ddf79c97aa25aa1fa05fe4b9ad6cd58d!